pages/workouts.js

```diff
@@ -1,19 +1,49 @@
 import AddWorkout from "@/components/AddWorkout/AddWorkout";
-import BackButtton from "@/components/BackButton/BackButtton";
-import ExerciseList from "@/components/ExerciseList/ExerciseList";
+import Footer from "@/components/Footer/Footer";
+import PageTitle from "@/components/Title/Title";
 import WorkoutList from "@/components/WorkoutList/WorkoutList";
-import Link from "next/link";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { useState } from "react";
+import styled from "styled-components";
 
 export default function HomePage() {
     const [visible, setVisible] = useState(false);
-    return (
-        <div>
-            <BackButtton />
-            <button onClick={() => setVisible(true)}>Add Workout</button>
-            {visible && <AddWorkout setVisible={setVisible}/>}
-            <h1>Workouts</h1>
-            <WorkoutList/>
-        </div>
+    return (<div>
+        <WorkoutsHeader>
+            <PageTitle text={'Workouts'}/>
+            <AddButton onClick={() => setVisible(true)}>
+                    <FontAwesomeIcon icon={faPlus} />
+            </AddButton>
+
+        </WorkoutsHeader>
+        {visible && <AddWorkout setVisible={setVisible} />}
+        <WorkoutList />
+        <Footer />
+    </div>
     );
 }
+
+const WorkoutsHeader = styled.div`
+padding: 20px;
+display: flex;
+justify-content: space-between;
+`
+
+const AddButton = styled.button`
```

Comment:
It looks like the `AddButton` is identical to the `SubmitButton` in your `AddWorkout` component. Instead of duplicating the component, consider creating a single reusable button component that can be used in multiple places. This will reduce redundancy, make your code easier to maintain, and ensure consistent styling and behavior across your application.